frontend/gulpfile.js

```diff
@@ -8,6 +8,7 @@ var rename = require('gulp-rename');
 var sh = require('shelljs');
 var inject = require('gulp-inject');
 var watch = require('gulp-watch');
+var Highcharts = require('highcharts');
```

Comment:
No idea why highcharts is required here, but should for sure not be needed in gulp file.